### PR TITLE
namespace description

### PR DESCRIPTION
**Pull request by fahd-akbar**

### CODE REVIEW

File: com.regnosys.rosetta/src/com/regnosys/rosetta/generator/RosettaGenerator.xtend [hunk 1]
```diff
@@ -72,8 +77,13 @@ class RosettaGenerator extends AbstractGenerator {
 		try {
 			lock.getWriteLock(true);
 			if (!ignoredFiles.contains(resource.URI.segments.last)) {
+				
+				var rosettaModelList = resource.contents.filter(RosettaModel)
+				var cdmVersion  = rosettaModelList.get(0).version // pick version from the first model
```
Comment (minesh-s-patel) [inline com.regnosys.rosetta/src/com/regnosys/rosetta/generator/RosettaGenerator.xtend]: Dont use word CDM. CDM is a downstream model

Comment (minesh-s-patel) [inline com.regnosys.rosetta/src/com/regnosys/rosetta/generator/RosettaGenerator.xtend]: Why do we choose the first from the list here?

File: com.regnosys.rosetta/src/com/regnosys/rosetta/generator/java/object/JavaPackageInfoGenerator.xtend [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+	private def generatePackageInfo(String packageName, Collection<String> descriptions, String version) {
+		return '''
+			/**
+			*	«FOR desc : descriptions»
+			*	«desc»
+			*	<p>
+			*	«ENDFOR»
+			*
+			*	@author ISDA CDM
```
Comment (minesh-s-patel) [inline com.regnosys.rosetta/src/com/regnosys/rosetta/generator/java/object/JavaPackageInfoGenerator.xtend]: Do not add ISDA CDM.

The rosetta dsl project has nothing to do with CDM.

File: com.regnosys.rosetta/src/com/regnosys/rosetta/generator/java/util/ModelNamespaceUtil.xtend [hunk 3]
```diff
@@ -0,0 +1,20 @@
+package com.regnosys.rosetta.generator.java.util
+
+import com.google.common.collect.LinkedHashMultimap
+import com.regnosys.rosetta.rosetta.RosettaModel
+import java.util.List
+
+class ModelNamespaceUtil {
```
Comment (minesh-s-patel) [inline com.regnosys.rosetta/src/com/regnosys/rosetta/generator/java/util/ModelNamespaceUtil.xtend]: Make it public if you want to use this anywhere esle.

Comment (fahd-akbar) [reply com.regnosys.rosetta/src/com/regnosys/rosetta/generator/java/util/ModelNamespaceUtil.xtend]: public modifier is not needed in xtend files apparently :)

**Pull request merged**